docs/content/ecs/world-commands.md

```diff
@@ -0,0 +1,126 @@
+
+    protected override void OnUpdate()
+    {
+        for(var i = 0; i < data.Length; i++)
+        {
+            var requests = data.CreateEntitySender[i].RequestsToSend;
+            var entity = CreatureTemplate.CreateCreatureEntityTemplate(
+                new Coordinates(0, 0, 0));
+
+            requests.Add(new WorldCommands.CreateEntity.CreateRequest
+            (
+                entity
+            ));
+
+            data.CreateEntitySender[i].requestsd;
+        }
+    }
+}
+```
+
+This system iterates through every entity with a `Foo` component and sends a create entity request.
+
+### 3.  Delete an entity
+
+You can delete entities via the `DeleteEntity` world command. You need to know the SpatialOS entity ID of the entity you want to delete.
+
+To send a request use a `WorldCommands.DeleteEntity.CommandSender` component. This contains a list of `WorldCommands.DeleteEntity.Request` structs. Add a struct to the list to send the command.
+
+- `TimeoutMillis` is optional.
+
+To receive a response use `WorldCommands.DeleteEntity.CommandResponses`. This contains a list of `WorldCommands.DeleteEntity.ReceivedResponse`.
+
+```csharp
+public class DeleteCreatureSystem : ComponentSystem
+{
+    public struct Data
+    {
+        public readonly int Length;
+        [ReadOnly] public ComponentDataArray<Bar> Bar;
+        public ComponentDataArray<WorldCommands.DeleteEntity.CommandSender> DeleteEntitySender;
+        [ReadOnly] public ComponentDataArray<SpatialEntityId> SpatialEntityIds;
+    }
+
+    [Inject] Data data;
+
+    protected override void OnUpdate()
+    {
+        for(var i = 0; i < data.Length; i++)
+        {
+            var requests = data.DeleteEntitySender[i].RequestsToSend;
+            var entityId = data.SpatialEntityIds[i].EntityId;
+        	
+            requests.Add(new WorldCommands.DeleteEntity.CreateRequest
+            (
+                entityId
+            ));
+
+            data.DeleteEntitySender[i].RequestsToSend = requests;
+        }
+    }
+}
+```
+
+This system iterates through every entity with a `Bar` component and sends a delete entity request.
+
+### 4. Entity query
+
+You can use entity queries to get information about entities in the world. 
+
+To send a request use a `WorldCommands.EntityQuery.CommandSender` component. This contains a list of `WorldCommands.EntityQuery.Request` structs. Add a struct to the list to send the command.
+
+- For more information, see [entity queries](https://docs.improbable.io/reference/latest/shared/glossary#queries) in the SpatialOS documentation.
+- `TimeoutMillis` is optional.
+
+To receive a response use `WorldCommands.EntityQuery.CommandResponses`. This contains a list of `WorldCommands.EntityQuery.ReceivedResponse`.
+
+**Warning**: Entity queries only exist in the GDK in a prototype form, the methods for sending and receiving them exist, but you can only access some information  safely in the responses. It is not recommended to use them.
```

Comment:
It is not recommended to use them **at this time**. 

Makes it clear that we are intending to fix them